src/fluent-api/tests/click/clcik.msedge.headfull.test.ts

```diff
@@ -0,0 +1,186 @@
+import * as SUT from '../../playwright-fluent';

```

Comment:
There is a typo in the filename : `clcik.msedge.headfull.test.ts` it should be `click.msedge.headfull.test.ts`